g3w-admin/client/views.py

```diff
@@ -82,8 +82,13 @@ def get_context_data(self, **kwargs):
         contextData = super(ClientView, self).get_context_data(**kwargs)
 
         # group serializer
+        try:
+            group = self.project.group
+        except:
+            group = get_object_or_404(Group, slug=kwargs['group_slug'])
+
         groupSerializer = GroupSerializer(
-            getattr(self.project, 'group', get_object_or_404(Group, slug=kwargs['group_slug'])),
+            group,
```

Comment:
@wlorenzetti I tried it like the following and it works the same:


```py
# group serializer
# try:
#     group = self.project.group
# except:
#     group = get_object_or_404(Group, slug=kwargs['group_slug'])

groupSerializer = GroupSerializer(
    self.project.group,
    projectId=str(self.project.pk),
    projectType=kwargs['project_type'],
    request=self.request
)
```

Just to know, the CU plugin is a special case of projects without a group?

Reply:
@Raruto do you mean CDU?

Reply:
@wlorenzetti maybe it's a mispelling, I find it through git blame: https://github.com/g3w-suite/g3w-admin/commit/e634b10485301240f8cc28923a55f508393324eb (commited on Nov 27, 2017)